cdc/processor/processor.go

```diff
@@ -367,7 +367,7 @@ func (p *processor) handleTableOperation(ctx cdcContext.Context) error {
 				})
 			case model.OperProcessed:
 				if table.Status() != tablepipeline.TableStatusStopped {
-					log.Debug("the table is still not stopped", zap.Uint64("checkpointTs", table.CheckpointTs()), zap.Int64("tableID", tableID))
+					log.Info("the table is still not stopped", zap.Uint64("checkpointTs", table.CheckpointTs()), zap.Int64("tableID", tableID))
```

Comment:
Could you also change Debug to Info for the log at line 734?

https://github.com/pingcap/ticdc/blob/4f7c0b9606bc2c97f793caac42970aecc8f0a6c1/cdc/processor/processor.go#L734

Reply:
I'm afraid there are too many logs if table count is large, such as 10k tables.

Reply:
I also revert this log level change, since
1. we can deduce lazy stop from sink barrier block log
2. It is better to refine scheduling logs in a specific PR